cryptus-app/styles/_variables.scss

```diff
@@ -22,3 +22,6 @@ $secondary-color-pale: #eee2e5;
 
 $border-radius-m: 5px;
 $border-radius-l: 10px;
+
+$collection_height: 15%;
+$overview-height: 100%
```

Comment:
don't use % in global variables. If it really is 100% write it in the css instead, if it should be in px, write in px in global